core/domain/skill_services.py

```diff
@@ -915,11 +1041,18 @@ def compute_summary_of_skill(
 
     Returns:
         SkillSummary. The computed summary for the given skill.
+
+    Raises:
+        Exception. No data available for when the skill was last_updated.
     """
     skill_model_misconception_count = len(skill.misconceptions)
     skill_model_worked_examples_count = len(
         skill.skill_contents.worked_examples)
 
+    if skill.created_on is None or skill.last_updated is None:
+        raise Exception(
+            'No data available for when the skill was last_updated.'
```

Comment:
In the event that skill.created_on is the one that is None, is this error message accurate?

Reply:
Fixed